dissect/target/plugins/os/unix/log/messages.py

```diff
@@ -1,37 +1,58 @@
-import bz2
-import datetime
 import re
-import zlib
-from io import StringIO
+from datetime import datetime, timezone
+from itertools import chain
+from typing import Generator
+from zoneinfo import ZoneInfo, ZoneInfoNotFoundError
 
+from dissect.util import ts
+from flow.record.fieldtypes import path
+
+from dissect.target.helpers.fsutil import TargetPath, open_decompress
 from dissect.target.helpers.record import TargetRecordDescriptor
 from dissect.target.plugin import Plugin, export
 
 MessagesRecord = TargetRecordDescriptor(
     "linux/messages",
     [
         ("datetime", "ts"),
-        ("uri", "filepath"),
         ("string", "daemon"),
         ("varint", "pid"),
-        ("string", "msg"),
+        ("string", "message"),
         ("string", "raw"),
+        ("path", "source"),
     ],
 )
 
-RE_TS = re.compile(r"(\w+\s\d+\s\d{2}:\d{2}:\d{2})")
+RE_TS = re.compile(r"(\w+\s{1,2}\d+\s\d{2}:\d{2}:\d{2})")
 RE_DAEMON = re.compile(r"^[^:]+:\d+:\d+[^\[\]:]+\s([^\[:]+)[\[|:]{1}")
 RE_PID = re.compile(r"\w\[(\d+)\]")
 RE_MSG = re.compile(r"[^:]+:\d+:\d+[^:]+:\s(.*)$")
 
 
 class MessagesPlugin(Plugin):
+    def __init__(self, target):
+        super().__init__(target)
+
+        try:
+            self.target_timezone = ZoneInfo(f"{target.timezone}")
+        except ZoneInfoNotFoundError:
+            self.target.log.warning("Could not determine timezone of target, falling back to UTC.")
+            self.target_timezone = timezone.utc
+
     def check_compatible(self):
-        pass
+        return self.target.fs.path("/var/log/syslog").exists() or self.target.fs.path("/var/log/messages").exists()
```

Comment:
```suggestion
        return any(self.target.fs.path("/var/log").glob("syslog*")) or any(self.target.fs.path("/var/log").glob("messages*"))
```
Might be more accurate, and `any` will ensure it'll return as quickly as possible.